draft-ietf-rats-corim.md

```diff
@@ -825,6 +825,21 @@ $version-scheme /= int / text
 
 [^issue] https://github.com/ietf-rats-wg/draft-ietf-rats-corim/issues/8
 
+The following describes details the security version number (svn) and the minimum security version number (min-svn).
```

Comment:
There seems to be one or more missing piece(s) of English language here, but I can't tell exactly what :)

* The following describes _in_ details the security [...]
* The following details the security [...]
* The following describes _the_ details _of_ the security [...]

Reply:
Ned, if you address this we can merge.